charts/sourcegraph/values.yaml

```diff
@@ -577,21 +810,43 @@ searcher:
       memory: 500M
 
 storageClass:
-  create: true # Disable if you have your own existing storage class
-  name: sourcegraph # Customize to the existing storage class name
+  # -- Enable creation of storageClass
+  # Disable if you have your own existing storage class
```

Comment:
The formatting in the chart ends up weird here and in the other storage class descriptions - i.e. ```Enable creation of storageClass Disable if you have your own existing storage class```

Do we need to add a period on the top line or is there a better solution?

Reply:
adding a `period` works!